portal-impl/src/com/liferay/portal/upgrade/v7_4_x/PortalUpgradeProcessRegistryImpl.java

```diff
@@ -620,6 +620,15 @@ protected String[][] getTableAndPrimaryKeyColumnNames() {
 				"activitySettingId asc"),
 			new DeleteDuplicateUniqueFinderRowsUpgradeProcess(
 				"Ticket", new String[] {"key_"}, "ticketId asc"));
+
+		upgradeVersionTreeMap.put(
+			new Version(32, 2, 0),
```

Comment:
Why there is a jump from 32.0.0 to 32.2.0?

Reply:
SF 🤷🏻‍♀️